### DISCUSSION TITLE
在手动设置progress的情况下，pag播放会出现刷新不及时、卡顿现象

### DISCUSSION THREAD

**Discussion by zz6123816** [Tencent/libpag#2359] [Q&A] [answered]
在手动设置progress的场景下，若即将设置的 nextProgress 小于 currentProgress 时，会大概率出现卡顿现象，请问这种场景在使用上该如何适配？

**Comment by domchen** [comment, marked as answer]
这是类似倒放的场景，如果你遇到倒放会卡顿的情况，那么一定是素材里出现了视频序列帧，也就是用到了BMP预合成导出。去掉这个用纯矢量的素材就不会存在顺序问题。怎么播放都不会卡。视频天然就是要正序播放的。

**Reply by zz6123816** [reply]
如果使用 PAGDecoder / PAGImageView 可以解决此问题吗？看描述像是专门针对序列帧做适配的API，我们试了下在新版 pag 中好像就没有之前那么卡了

**Reply by domchen** [reply]
对，缓存了就可以解决的。
